st2actions/st2actions/notifier/notifier.py

```diff
@@ -79,22 +78,18 @@ def process(self, execution_db):
         extra = {'execution': execution_db}
         LOG.debug('Processing action execution "%s".', execution_id, extra=extra)
 
-        if execution_db.status not in LIVEACTION_COMPLETED_STATES:
```

Comment:
Not a blocker, but I found previous code a bit easier to read / follow ("return early" vs "nested if", one level less of nesting).

Was there any particular reason why you changed it?

Reply:
This change is inevitable since:

1. I thought the order of other function calls is important here and shouldn't be changed (to keep this: `apply_post_run_policies` -> `_post_notify_triggers` -> `_post_generic_trigger`)
2. We always need to call `_post_generic_trigger` regardless of the status. So If we want to do a "early return" then I need to move that call before `if execution_db.status not in LIVEACTION_COMPLETED_STATES:` line but that will break the order (`_post_generic_trigger` will be called before `_post_notify_triggers`)